tic-tac-commandline.py

```diff
@@ -0,0 +1,206 @@
+            self.game_round+=1
+        if picked_slot==3 and self.grid_slot3==2 and self.active_player==1:
+            self.grid_slot3=1
+            self.active_player=0
+            self.game_round+=1
+        if picked_slot==4 and self.grid_slot4==2 and self.active_player==1:
+            self.grid_slot4=1
+            self.active_player=0
+            self.game_round+=1
+        if picked_slot==5 and self.grid_slot5==2 and self.active_player==1:
+            self.grid_slot5=1
+            self.active_player=0
+            self.game_round+=1
+        if picked_slot==6 and self.grid_slot6==2 and self.active_player==1:
+            self.grid_slot6=1
+            self.active_player=0
+            self.game_round+=1
+        if picked_slot==7 and self.grid_slot7==2 and self.active_player==1:
+            self.grid_slot7=1
+            self.active_player=0
+            self.game_round+=1
+        if picked_slot==8 and self.grid_slot8==2 and self.active_player==1:
+            self.grid_slot8=1
+            self.active_player=0
+            self.game_round+=1
+        if picked_slot==9 and self.grid_slot9==2 and self.active_player==1:
+            self.grid_slot9=1
+            self.active_player=0
+            self.game_round+=1
+        
+        #for player 2 turn
+        if picked_slot==1 and self.grid_slot1==2 and self.active_player==0:
+            self.grid_slot1=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==2 and self.grid_slot2==2 and self.active_player==0:
+            self.grid_slot2=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==3 and self.grid_slot3==2 and self.active_player==0:
+            self.grid_slot3=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==4 and self.grid_slot4==2 and self.active_player==0:
+            self.grid_slot4=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==5 and self.grid_slot5==2 and self.active_player==0:
+            self.grid_slot5=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==6 and self.grid_slot6==2 and self.active_player==0:
+            self.grid_slot6=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==7 and self.grid_slot7==2 and self.active_player==0:
+            self.grid_slot7=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==8 and self.grid_slot8==2 and self.active_player==0:
+            self.grid_slot8=0
+            self.active_player=1
+            self.game_round+=1
+        if picked_slot==9 and self.grid_slot9==2 and self.active_player==0:
+            self.grid_slot9=0
+            self.active_player=1
+            self.game_round+=1
+        self.check_for_winner()
+            
+    def check_for_winner(self):
+        print("checking for winner...")
```

Comment:
For improved server-side debugging, it is recommended to replace prints with logging.